raisinbread/README.md

```diff
@@ -50,12 +50,21 @@ cat raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql
 
 ***Important:** Ensure that the above commands were completed properly and that all the tables were dropped before continuing the installation process.*
 
+If you could not drop all the tables with the commands above, you have the option to drop the tables by manually sourcing these .sql files directly 
```

Comment:
I would specify here that these tables need to be dropped in this order. If you try to source "9999-99-99-drop_tables.sql" first, there will be errors.

Reply:
Thanks for the comment. I have added a note here.